packages/block-editor/src/components/grid/grid-visualizer.js

```diff
@@ -146,56 +144,117 @@ function ManualGridVisualizer( { gridClientId, gridInfo } ) {
 		return rects;
 	}, [ gridItems ] );
 
-	return range( 1, gridInfo.numRows ).map( ( row ) =>
-		range( 1, gridInfo.numColumns ).map( ( column ) => {
-			const isCellOccupied = occupiedRects.some( ( rect ) =>
-				rect.contains( column, row )
-			);
-			const isHighlighted =
-				highlightedRect?.contains( column, row ) ?? false;
-			return (
-				<GridVisualizerCell
-					key={ `${ row }-${ column }` }
-					color={ gridInfo.currentColor }
-					className={ isHighlighted && 'is-highlighted' }
-				>
-					{ isCellOccupied ? (
-						<GridVisualizerDropZone
-							column={ column }
-							row={ row }
-							gridClientId={ gridClientId }
-							gridInfo={ gridInfo }
-							setHighlightedRect={ setHighlightedRect }
-						/>
-					) : (
-						<GridVisualizerAppender
-							column={ column }
-							row={ row }
-							gridClientId={ gridClientId }
-							gridInfo={ gridInfo }
-							setHighlightedRect={ setHighlightedRect }
-						/>
-					) }
-				</GridVisualizerCell>
-			);
-		} )
+	return (
+		<BlockPopoverCover
+			__unstablePopoverSlot="__unstable-block-tools-after"
+			className={ clsx( 'block-editor-grid-visualizer', {
```

Comment:
The class names in this file have come out of sync with the component names. It used to be that `GridVisualizer` rendered an element with `className = 'block-editor-grid-visualizer'` and so on. Should we update them?

Also, is it bad that both `BlockPopover` and `BlockPopunder` render an element with the same classname?

Reply:
Good point. I've pushed a commit that updates these. `GridPopunder` doesn't need classnames it relies on the inline css, so removing those solves the duplication. I changed the `GridPopover` classnames to `block-editor-grid-visualizer-popover` to match the components. 

Also removed another unused classname.

It's not perfect, but I think good enough right now.